judgments/models.py

```diff
@@ -119,3 +124,87 @@ class Meta:
         namespaces = {"search": "http://marklogic.com/appservices/search"}
 
     transform_to_html = xmlmodels.XsltField(join(dirname(__file__), "search_match.xsl"))
+
+
+class CourtDates(models.Model):
+    param = models.CharField(max_length=64, primary_key=True)
+    start_year = models.IntegerField(blank=False)
+    end_year = models.IntegerField(blank=False)
+
+
+class CourtDecoratedWithDates:
```

Comment:
Are there good reasons for this not to be a subclass of utils.Court?

Reply:
none at all - and that will simplify things nicely. Will do that now! 🙌